tests/test_list_objects.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
Can you add the below test in the `test_upload.py` file instead of creating a new file?

Reply:
merged to test_upload.py.